core/src/main/java/org/lflang/TargetConfig.java

```diff
@@ -389,7 +389,9 @@ public static class PlatformOptions {
     /**
      * The string value used to determine what type of embedded board we work with and can be used
      * to simplify the build process. For example, when we want to flash to an Arduino Nano 33 BLE
-     * board, we can use the string arduino:mbed_nano:nano33ble
+     * board, we can use the string arduino:mbed_nano:nano33ble Can also be used to specify uart
+     * output setting on rp2040 boards where arduino_nano_rp2040_connect:uart or
+     * arduino_nano_rp2040_connect:usb (usb) fully specifies the board and uart output
```

Comment:
This is unclear. First, the string arduino:mbed_nano:nano33ble is probably not right in a target property because it will not be valid syntax (too many colons). Also, isn't this specifying where stdout is directed rather than a "uart output setting"?